dongamboard/app/src/main/java/com/study/dongamboard/PostActivity.kt

```diff
@@ -12,35 +12,32 @@ import android.widget.Toast
 import androidx.appcompat.app.AlertDialog
 import androidx.appcompat.app.AppCompatActivity
 import com.study.dongamboard.adapter.CommentAdapter
+import com.study.dongamboard.api.ApiObject
 import com.study.dongamboard.db.CommentDB
+import com.study.dongamboard.db.PostDB
 import com.study.dongamboard.model.CommentData
 import com.study.dongamboard.model.PostData
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
 
 class PostActivity : AppCompatActivity() {
 
+    lateinit var postDB: PostDB
+    lateinit var post: PostData
```

Comment:
response dto 역할이면 아래 처럼 이름을 바꾸는건 어떠신가요!
```suggestion
    lateinit var post: PostResponse
```

Reply:
았 확실히 Response type에 가까운 것 같습니다..(이름을 적당히 지어놓은 자) 감사합니다!